docs/docs/webchat/webchat-enable.md

```diff
@@ -0,0 +1,34 @@
+---
+id: webchat-enable
+title: Enabling and Disabling
```

Comment:
I would name this as `Enabling and disabling webchat visibility` or similar. Ow it sounds pretty generic

Reply:
Title changed.